### PR TITLE
Create ZOON Radio

### PR DESCRIPTION
**Pull request by artemskorsky**
added ZOON page

### CODE REVIEW

File: en/telemetry/ZOON Radio [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ZOON Telemetry Radio
+
+Dronee produce  2.4GHz Long Range Telemetry module with Lora Technology which has Mesh Technology. [SiK](../telemetry/sik_radio.md)-compatible telemetry radios that can be used with PX4. 
+2.4GHz
+![SiK Radio](../../assets/hardware/telemetry/holybro_sik_radio.jpg)
```
Comment (hamishwillee) [inline en/telemetry/ZOON Radio:5]: Image is of Holybro radio, not yours.

File: en/telemetry/ZOON Radio [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# ZOON Telemetry Radio
+
+Dronee produce  2.4GHz Long Range Telemetry module with Lora Technology which has Mesh Technology. [SiK](../telemetry/sik_radio.md)-compatible telemetry radios that can be used with PX4.
```
Comment (hamishwillee) [inline en/telemetry/ZOON Radio:3]: Typo "Dronee"

**Comment by hamishwillee** [comment]
Thanks. Please rename the file to **dronee_zoon_sik_radio.md**. You will also need to edit /en/SUMMARY.md file so that this gets included in the sidebar table of contents.

**Comment by artemskorsky** [comment]
> Thanks. Please rename the file to **dronee_zoon_sik_radio.md**. You will also need to edit /en/SUMMARY.md file so that this gets included in the sidebar table of contents.
> 
> I gave this a little subedit, but it still need an image of your radio, not the Holybro one.

i have added. but i am not sure i did it correctly. i am not familiar with github

**Comment by hamishwillee** [comment]
Good job. Can you please replace the current image with this one - it will make the page look a lot better once rendered and is much smaller/faster to load. 

ZOON_Telemetry

File: en/SUMMARY.md [hunk 3]
```diff
@@ -152,9 +152,10 @@
       * [Trimble MB-Two](gps_compass/rtk_gps_trimble_mb_two.md)
   * [Telemetry Radios](telemetry/README.md)
     * [SiK Radio](telemetry/sik_radio.md)
-      * [RFD900 (SiK) Telemetry Radio](telemetry/rfd900_telemetry.md)
+      * [Dronee ZOON 2.4Ghz Lora (SiK) Telemetry Radio](telemetry/dronee_zoon_sik_radio.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:155]: PS why removing the RFD900?

**Comment by hamishwillee** [comment]
I am closing this. Feel free to reopen if you're able to provide the required information.

**Pull request closed without merging**